pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1897,9 +1897,12 @@ func (s *TracerSuite) TestGetHelpersTelemetry() {
 	}
 
 	// Ensure `bpf_probe_read_user` fails by passing an address guaranteed to pagefault to open syscall.
-	_, _, sysErr := syscall.Syscall6(syscall.SYS_MMAP, uintptr(0xdeadbeef), uintptr(syscall.Getpagesize()), syscall.PROT_READ|syscall.PROT_WRITE, syscall.MAP_ANON|syscall.MAP_PRIVATE, 0, 0)
+	addr, _, sysErr := syscall.Syscall6(syscall.SYS_MMAP, uintptr(0), uintptr(syscall.Getpagesize()), syscall.PROT_READ|syscall.PROT_WRITE, syscall.MAP_ANON|syscall.MAP_PRIVATE, 0, 0)
```

Comment:
Because the kernel may not actually return the page mapped at `0xdeadbeef` if a mapping already exists at this address.
Since we cannot be sure that the page was mapped at the requested address, we may as well just let the kernel decide and use the returned address.